Bottom: f6e20f2cd10ce9d7b3d48b4a2000923e0d815581
Top:    8e5aca86651b7f41970ce487f2233a78fcf7e1d3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-29 02:40:14 +0100

Refresh of add-vrml-model-rendering-suppo

---

diff --git a/src/hid/common/hidgl_package_acy_resistor.c b/src/hid/common/hidgl_package_acy_resistor.c
index 412668e..53fd75b 100644
--- a/src/hid/common/hidgl_package_acy_resistor.c
+++ b/src/hid/common/hidgl_package_acy_resistor.c
@@ -1070,7 +1070,7 @@ hidgl_draw_acy_resistor (ElementType *element, float surface_depth, float board_
   glActiveTextureARB (GL_TEXTURE0_ARB);
   glGenTextures (1, &texture1);
   glBindTexture (GL_TEXTURE_1D, texture1);
-  zero_ohm = setup_resistor_texture (element, resistor_body_color);
+  zero_ohm = setup_resistor_texture (element, resistor_body_color, 3);
 
   glTexEnvf (GL_TEXTURE_ENV, GL_TEXTURE_ENV_MODE, GL_MODULATE);
   glTexParameterf (GL_TEXTURE_1D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
@@ -2379,7 +2379,6 @@ hidgl_draw_1650mil_cap (ElementType *element, float surface_depth, float board_t
 
   first_run = false;
 }
-<<<<<<< current
 
 void
 hidgl_draw_350x800mil_cap (ElementType *element, float surface_depth, float board_thickness)
diff --git a/src/hid/common/hidgl_package_vrml_l.l b/src/hid/common/hidgl_package_vrml_l.l
index eda9a78..6a27388 100644
--- a/src/hid/common/hidgl_package_vrml_l.l
+++ b/src/hid/common/hidgl_package_vrml_l.l
@@ -189,7 +189,7 @@ vector			{ return T_vector; }
 							 * stringlength is counted and copied without
 							 * leading and trailing '"'
 							 */
-						vrml_yylval.string = MyCalloc(vrml_yyleng+1, sizeof(char), "LEX");
+						vrml_yylval.string = calloc (vrml_yyleng+1, sizeof(char), "LEX");
 						p1 = (char *) (vrml_yytext);
 						p2 = vrml_yylval.string;
 						while(vrml_yyleng--)
